docs/cloud/sso.md

```diff
@@ -0,0 +1,133 @@
+
+lakeFS Cloud and lakeFS Enterprise support the commonly used protocols for authentication, such as OpenID Connect and SAML.
+
+There are specific authentication utilities that we've already verified that are working, if you're using one that is not listed below but is based on OpenID Connect or SAML, it's probably a small effort for us to make sure it'll work, please [contact us](support@treeverse.io) for more information.
+
+## Cloud
+
+lakeFS Cloud leverage Auth0's capabilities for authentication, therefore, we're supporting Auth0's identity providers, such as: Active Directory/LDAP, ADFS, Azure Active Directory Native, Google Workspace, OpenID Connect, Okta, PingFederate, SAML and Azure Active Directory.
+
+### Okta
+
+> **Note**: this guide is based on [Okta's Create OIDC app integrations guide](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm).
+
+Steps:
+1. Login to your Okta account
+2. Select **Applications > Applications**, then **Create App Integration**.
+3. Select Create New App and enter the following:
+    1. For **Sign-in method**, choose OIDC.
+    2. Under **Application type**, choose **Web app**.
+    3. Select Next.
+4. Under General Settings:
+    1. **App integration name**, enter a name for your application. (i.e lakeFS Cloud)
+5. In the **Sign-in redirect URIs** field, enter https://lakefs-cloud.us.auth0.com/login (United States) or https://lakefs-cloud.eu.auth0.com/login (Europe).
+6. Under **Sign-in redirect URIs**, click **Add URI**, enter https://lakefs-cloud.us.auth0.com/login/callback (United States) or https://lakefs-cloud.eu.auth0.com/login/callback (Europe).
+7. Under **Assignments**, choose the wanted **Controlled access**. (i.e Allow everyone in your organization to access)
+8. Uncheck **Enable immediate access with Federation Broker Mode**.
+9. Select **Save**.
+
+Once you finish registering your application with Okta, save the **Client ID**, **Client Secret** and your **Okta Domain**, send this to Treeverse's team to finish the integration.
+
+
+### Active Directory Federation Services (AD FS)
+
+Prerequisites:
+* Client's AD FS server should be exposed publicly or to Auth0's IP ranges (either directly or using Web Application Proxy)
+
+Steps:
+1. Connect to the AD FS server
+2. Open AD FS' PowerShell CLI as Administrator through the server manager
+3. Execute the following:
+
+    ```sh
+    (new-object Net.WebClient -property @{Encoding = [Text.Encoding]::UTF8}).DownloadString("https://raw.github.com/auth0/adfs-auth0/master/adfs.ps1") | iex
+
+    AddRelyingParty "urn:auth0:lakefs-cloud" "https://lakefs-cloud.us.auth0.com/login/callback"
+    ```
+
+    **Note**: If your organization data is located in Europe, use `lakefs-cloud.eu.auth0.com` instead of `lakefs-cloud.us.auth0.com`.
+
+Once you finish registering lakeFS Cloud with AD FS, save the **AD FS URL** and send this to Treeverse's team to finish the integration.
+
+### Azure AD
```

Comment:
```suggestion
## Azure Active Directory (AD)
```

Reply:
We should be consistent with acronyms and expand them at first use (but also retain them to help with search if ppl would always look for "AD")